.github/actions/run-cypress-tests/action.yml

```diff
@@ -55,14 +59,22 @@ runs:
       if: ${{ runner.os == 'Linux' }}
       run: |
         cd ./OpenSearch-Dashboards/plugins/security-dashboards-plugin
-        yarn runIdp
+        if [ -z "${{ inputs.osd_base_path }}" ]; then
+          yarn runIdp &
+        else
+          yarn runIdp --basePath ${{ inputs.osd_base_path }} &
+        fi
```

Comment:
this can be shortened to `yarn runIdp --basePath ${{ inputs.osd_base_path }} &` since empty basePath would not affect the URL.

Reply:
Nice, this sounds pretty clean!

Reply:
If osd_base_path is empty then the command run is `yarn runIdp --basePath &` and minimist parses --basePath as a boolean flag instead of an empty string.